synapse/handlers/e2e_keys.py

```diff
@@ -283,14 +283,26 @@ def get_cross_signing_keys_from_cache(self, query, from_user_id):
         self_signing_keys = {}
         user_signing_keys = {}
 
-        # Currently a stub, implementation coming in https://github.com/matrix-org/synapse/pull/6486
-        return defer.succeed(
-            {
-                "master_keys": master_keys,
-                "self_signing_keys": self_signing_keys,
-                "user_signing_keys": user_signing_keys,
-            }
-        )
+        user_ids = list(query)
+
+        keys = yield self.store.get_e2e_cross_signing_keys_bulk(user_ids, from_user_id)
```

Comment:
Oh, this function now needs a `@defer.inlineCallbacks`

Reply:
Ah, I see.  Neil removed the `@defer.inlineCallbacks` when he stubbed out the function, and I missed it when I did the merge.  (Git complained about the conflict in the function body, but there was no conflict in the removed `@defer.inlineCallbacks`, so git didn't complain about that.)